class/defaults.yml

```diff
@@ -1,4 +1,86 @@
 parameters:
   appuio_cloud_reporting:
     =_metadata: {}
-    namespace: syn-appuio-cloud-reporting
+    namespace: appuio-cloud-reporting
+
+    images:
+      reporting:
+        registry: ghcr.io
+        repository: appuio/appuio-cloud-reporting
+        tag: v0.4.1
+
+      erp_adapter:
+        registry: ghcr.io
+        repository: vshn/appuio-odoo-adapter
+        tag: v1.0.3
+
+    schedules:
+      backfill: '15 * * * *'
+      check_missing: '30 8-14 * * *'
+      invoice: '0 10 1 * *'
+
+    database:
+      host: appuio-cloud-reporting
+      port: 5432
+      name: appuio-cloud-reporting
+      username: appuio-cloud-reporting
+      password: null
+      parameters: sslmode=disable
```

Comment:
Very small nitpick: The defaults for the database currently point to a DB running in the namespace managed by the component, but the component doesn't manage the DB. I find that a bit weird. I would probably prefer an obviously external host name like `db.example.com` (And not disable SSL by default)

On a related note, but definitely out of scope: Are we still using the YOLO DB for reporting? What is the plan there? I would somehow expect the component `appuio-cloud-reporting` to bring its own DB

Reply:
> I would probably prefer an obviously external host name like db.example.com

👍

> (And not disable SSL by default)

Good catch 😅

> Are we still using the YOLO DB for reporting?

Yes